File: MinVid/src/app/Models/comic.ts
export class Comic {
    id: string;
    name: string;
    description: string;
    artist: string;
    numberOfPage: number;
    tags: string[]; 
    uploadDate: Date;

    constructor(id: string, name: string, description: string, artist: string, numberOfPages: number, tags: string[]){
        this.id = id;
        this.name = name;
        this.description = description;
        this.artist = artist;
        this.numberOfPage = numberOfPages;
        this.tags = tags;
        this.uploadDate = new Date();
    }
}

File: MinVid-API/Models/Comic.cs
namespace MinVid_API.Models
{
    public class Comic
    {
        public string id { get; set; }
        public string name { get; set; }
        public string description { get; set; }
        public string artist { get; set; }
        public int numberOfPages { get; set; }
        public List<string> tags { get; set; }
        public DateTime uploadDate { get; set; }
    }
}
